Automattic/wp-calypso
# Universal Header: add it back for logged in users

[Context](p1687309187057579/1686531180.034329-slack-C03TY6J1A)

This PR adds the WPCOM nav bar back for logged in users on https://wordpress.com/support/

![image](https://github.com/Automattic/wp-calypso/assets/52076348/22d1e11e-e87a-41f2-bc49-2fc160693a89)

It takes into consideration the adminbar height when logged in, changing the `top` propriety of `x-dropdown` class.
 
## Testing
- Pull and run this branch
- Navigate to `cd apps/happy-blocks` and `run yarn dev --sync`
- Sandbox wordpress.com/support and visit the site
- WPCOM nav bar should be there for logged in and logged out users
